.github/workflows/deploy.yml

```diff
@@ -321,6 +330,7 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           GCP_GEOLOCATION_API_KEY: ${{ secrets.GCP_GEOLOCATION_API_KEY_PRODUCTION }}
           S3_BUCKET: ${{ github.ref == 'refs/heads/production' && vars.PRODUCTION_S3_BUCKET || vars.STAGING_S3_BUCKET }}
+          PYTHON: ${{ steps.setup-python.outputs.python-path }}
```

Comment:
This shouldn't be necessary, since it looks like `actions/setup-python` is already adding the specified version of `python` to the front of the `$PATH`